as2/3.0/modules/ROOT/pages/index.adoc

```diff
@@ -131,6 +131,17 @@ image::as2-sender-flow-demo.png[]
 
 image::as2-receive-flow-demo.png[]
 
+== Proxy Bypass Configuration
+
+To disable the proxy settings in Studio: 
+
+. In the Studio menu, select *Run as > Mule Application (configure)*. 
+. Select your app and navigate to the *Arguments* tab.
+. Add the following argument in *VM arguments* field: +
+`-Dhttp.nonProxyHosts=YOURSERVERURLHERETOBYPASS`
+
+Arguments accept multiple values, including wildcards, separated by a pipe (`|`), for example: `&#42;.local|127.0.0.1`
```

Comment:
Not important but there are a lot of parentheticals here. Can you make "For example:..." it's own sentence?